pmlc/dialect/abi/ir/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+pml_cc_library(
+  NAME ir
+  HDRS
+    dialect.h.inc
+    ops.cc.inc
+    ops.h.inc
+  SRCS
+    dialect.cc
+  DEPS
+    ::ops-gen
+  PUBLIC
+)
+
+pml_tblgen_library(
+  NAME ops-gen
+  TD_FILE ops.td
+  OPTS -dialect=comp
```

Comment:
This is probably wrong

Reply:
Oops -- thanks for the catch.  (What's the right way to test these?)